README.md

```diff
@@ -1,2 +1,48 @@
 # brew-buddy-coffee-companion
 Companion app for tracking daily brewing and logging data
+
+
+## Routes
+All routes are found at the `/api/` endpoint.
+
+### Origin
+A coffee origin includes where the coffee comes from, and the recommended brew method. All routes require an authorized user.
+
+* `POST -- /origin` - Creates a new origin.
+* `PUT -- /origin/:id` - Update an origin.
+* `GET -- /origin/all` - Returns an array of all origins
+* `GET -- /origin/:id` - Returns a single origin
+* `GET -- /origin/:id/method` - Returns the origin's recommended brew method. [NOT YET IMPLEMENTED]
+* `GET -- /api/origin/:id/entries` - Returns all entires containing the specified origin. [NOT YET IMPLEMENTED]
+* `DELETE -- /origin/:id` - Deletes an origin id
+
+### Method
+A brew method contains information and instructions about brewing coffee with that method. Information includes how long the coffee should be brewed, instructions for brewing and the ratio between coffee and water.
+
+* `POST -- /method` - Creates a new method.
```

Comment:
It would be nice to have example code of what each route expects and returns

eg: 
* expected headers
* expected body
* expected query 

// example response